rqd/tests/rqmachine_tests.py

```diff
@@ -348,11 +348,6 @@ def test_getTempPath(self, gettempdirMock):
 
         self.assertEqual('%s/' % tmpDir, self.machine.getTempPath())
 
-    def test_getTempPathMcp(self):
```

Comment:
Can we keep this test and update it to expect the updated `getTempPath` response?

Reply:
As the `getTempPath` had been simplified to calling `tempfile.gettempdir` on linux I thought it would be better to remove it as it would be unit testing the standard library. thoughts?

Reply:
@gregdenton would you still want to keep the test?